tests/load-tests/cluster_read_config.yaml

```diff
@@ -25,3 +25,15 @@
 - name: measurements.tekton_tekton_pipelines_controller_workqueue_depth
   monitoring_query: sum(tekton_pipelines_controller_workqueue_depth)
```

Comment:
Hello @gabemontero . Yesterday on the stand-up I was thinking about changing `..._workqueue_...` to `..._work_queue_...`, but was poking around it and now I think using this `sum(...)` causes it does not matter. So please ignore me.